velox/common/memory/Memory.h

```diff
@@ -600,13 +602,13 @@ template <typename Allocator, uint16_t ALIGNMENT>
 void* FOLLY_NULLABLE MemoryPoolImpl<Allocator, ALIGNMENT>::allocateZeroFilled(
     int64_t numMembers,
     int64_t sizeEach) {
-  VELOX_USER_CHECK_EQ(sizeEach, 1);
-  auto alignedSize = sizeAlign<ALIGNMENT>(ALIGNER<ALIGNMENT>{}, numMembers);
+  auto alignedSize =
+      sizeAlign<ALIGNMENT>(ALIGNER<ALIGNMENT>{}, numMembers * sizeEach);
```

Comment:
@xiaoxmeng 
This code tells me that the allocation is aligned (I might be wrong) and you added comment:
```
NOTE: 'allocateZeroFilled' memory allocation is not aligned.
```

Is this right?

Reply:
There is no alignment support for this API allocation. The reason that we do this alignment size calculation is mostly for consistent handling for the free path. We need a way to know how much memory has been reserved on allocation and do the same for free. There is no separate free API for this.